src/content/structured/components/divider/accessibility.mdx

```diff
@@ -0,0 +1,49 @@
+---
+path: "/components/divider/accessibility"
+
+date: "2024-12-02"
+
+title: "Divider"
+
+status: "BETA"
+
+subTitle: "Dividers are used to separate different elements in groups of content. Visually, they look like rules or lines."
+
+tabs:
+  [
+    { title: "Guidance", path: "/components/divider" },
+    { title: "Code", path: "/components/divider/code" },
+    { title: "Accessibility", path: "/components/divider/accessibility" },
+  ]
+---
+
+## Easy to use for everyone
+
+Horizontal dividers use the semantic `<hr>` ('thematic break' or 'horizontal rule') HTML element.
+
+Vertical dividers use the `separator` role and have an `aria-orientation` attribute set to `vertical`. These attributes are implicitly present on horizontal dividers (but with an `aria-orientation` of `horizontal`) because of the use of the `<hr>` element.
+
+## For Assistive Technology
+
+Labels on dividers will be announced by a screen reader when navigating with a virtual cursor.
+
+## Based on
+
+The divider component has been based on the following resources:
+
+<ul>
+  <li>
+    <ic-link
+      target="_blank"
+      href="https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/separator_role"
+      rel="noreferer noopener nofollow"
+    >
+      ARIA: separator role
+    </ic-link>
+    , accessed October 2024.
+  </li>
+</ul>
+
```

Comment:
Please can you add these links
https://www.digitala11y.com/separator-role/
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr